### PR TITLE
Feature/131022 trust overview re design

### PR DESCRIPTION
**Pull request by nimble-dfe**
**What is the change?**
Added sub navs
**Why do we need the change?**
Change for page load times
**What is the impact?**
UI
**Azure DevOps Ticket**
https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_boards/board/t/Concerns%20Casework/Stories/?workitem=131022

### CODE REVIEW

File: ConcernsCaseWork/ConcernsCaseWork/Pages/Trust/ClosedCases.cshtml.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace ConcernsCaseWork.Pages.Trust;
+
+public class ClosedCases : PageModel
```
Comment (mikestock-nimble) [inline ConcernsCaseWork/ConcernsCaseWork/Pages/Trust/ClosedCases.cshtml.cs:5]: QUESTION:

Are these files meant to be in this review?

File: ConcernsCaseWork/ConcernsCaseWork/Pages/Trust/Overview.cshtml [hunk 2]
```diff
@@ -7,23 +7,19 @@
 	ViewData["Title"] = "Trust Overview";
 	var nonce = HttpContext.GetNonce();
 }
+<style>
+.toggle-content {
```
Comment (mikestock-nimble) [inline ConcernsCaseWork/ConcernsCaseWork/Pages/Trust/Overview.cshtml:11]: SUGGESTION:

Can you move these styles into a global css file, we will probably need them for the other pagination changes we are going to make

We have ConcernsCaseWork\wwwroot\src\css\_extra.scss

File: ConcernsCaseWork/ConcernsCaseWork/Pages/Trust/Overview.cshtml [hunk 3]
```diff
@@ -40,22 +36,72 @@ else
...
+		</div>
+		
+		
+		
 	</div>
-}
\ No newline at end of file
+
+
+<script type="application/javascript" nonce="@nonce">
```
Comment (mikestock-nimble) [inline ConcernsCaseWork/ConcernsCaseWork/Pages/Trust/Overview.cshtml:82]: I think we also need these changes for the case management page

Case\Management\Index.cshtml

**Review by mikestock-nimble — CHANGES_REQUESTED** [review]

File: ConcernsCaseWork/ConcernsCaseWork/Pages/Trust/ClosedCases.cshtml.cs (hunk 1 above)
Comment (nimble-dfe) [reply ConcernsCaseWork/ConcernsCaseWork/Pages/Trust/ClosedCases.cshtml.cs]: removed

File: ConcernsCaseWork/ConcernsCaseWork/Pages/Trust/Overview.cshtml (hunk 2 above)
Comment (nimble-dfe) [reply ConcernsCaseWork/ConcernsCaseWork/Pages/Trust/Overview.cshtml]: Removed as not required

**Review by mikestock-nimble — APPROVED** [review]

**Pull request merged**